apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -186,6 +134,65 @@ export default function SectionsSetUpContainer({sectionToBeEdited}) {
     }
   };
 
+  const saveSection = section => {
+    // Determine data sources and save method based on new vs edit section
+    const dataUrl = isNewSection
+      ? SECTIONS_API
+      : `${SECTIONS_API}/${section.id}`;
+    const method = isNewSection ? 'POST' : 'PATCH';
+    const loginType = isNewSection
+      ? queryParams('loginType')
+      : section.loginType;
+    const participantType = isNewSection
+      ? queryParams('participantType')
+      : section.participantType;
+
+    const form = document.querySelector(`#${FORM_ID}`);
+    // If we find a missing field in the form, report which one and reset save status
+    if (!form.checkValidity()) {
+      form.reportValidity();
+      setIsSaveInProgress(false);
```

Comment:
This is the validation that catches when a user doesn't fill out a required field. The only change in this area: added the setIsSaveInProgress(false).